src/features/authorSlice.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-param-reassign */
```

Comment:
It's better to avoid using `eslint-disable` comments unless absolutely necessary. Try to fix the issue instead of ignoring it. If you really need to disable a rule, consider disabling it only for specific lines or rules, like `/* eslint-disable no-param-reassign */ 🛠.